.github/workflows/test.yaml

```diff
@@ -0,0 +1,78 @@
+name: test
+
+on:
+  pull_request:
+
+jobs:
+  generate:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
```

Comment:
maybe good to use go1.21?

Reply:
Ollama currently targets go1.20 for compatibility so that's what's being tested here